indico/modules/rb_new/client/js/common/bookings/BookingEdit.jsx

```diff
@@ -166,6 +170,11 @@ class BookingEdit extends React.Component {
             candidates = availabilityData.candidates;
         }
 
+        /* Filter out rejections and cancellations from candidates */
+        const {data: {cancellations, rejections}} = calendar;
```

Comment:
In code we usually use the american spelling (since it's much more common among programmers), so it's only one L in `cancellations`. Same in the CSS class etc.